packages/stratego-four/src/game/gameSaga.js

```diff
@@ -49,7 +66,127 @@ export function initializeWeb3(action) {
  */
 export function* initializeGame() {
   const accounts = yield select(selectAccounts);
-  yield initializePlayer(accounts[0]);
+
+  yield put(setPlayer(accounts[0]));
+}
+
+/**
+ * Initialize player pieces with default values
+ */
+export function* initializePlayerPieces() {
```

Comment:
...and I was wondering how the function caller would know they were actually getting a `Generator` type object/function....
Another TIL! [Good docs too 👍](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)

Reply:
![mind blown](https://media.giphy.com/media/OK27wINdQS5YQ/giphy.gif)